ai/core/src/unitycatalog/ai/core/utils/callable_utils.py

```diff
@@ -509,8 +548,80 @@ def generate_sql_function_body(
 
     metadata = extract_function_metadata(func)
 
+    sql_metadata = create_sql_metadata(metadata)
+
+    sql_body = assemble_sql_body(
+        catalog,
+        schema,
+        metadata.func_name,
+        sql_metadata.sql_params,
+        metadata.sql_return_type,
+        sql_metadata.func_comment,
+        sql_metadata.indented_body,
+        replace,
+    )
+
+    return sql_body
+
+
+def generate_wrapped_sql_function_body(
```

Comment:
Do we need this new function? we can make functions optional in generate_sql_function_body

Reply:
We don't explicitly, but my only concern was for some of the other feature request additions that the eval team will likely have in the next few weeks. Since everything they're doing for templating will require modifications specifically with function wrapping, I wanted to keep that edge logic external to the base standard path of building the function body statement. I'm simply worried about introducing too much complexity into a single utility function :)